extension-client/src/command/workspace/estimateCost.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * IBM Cloud Schematics
+ * (C) Copyright IBM Corp. 2021 All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ import * as vscode from 'vscode';
+import { estimateCost } from '../../command/shell/terraform/index';
+import EstimateCostView from '../../webview/workspace/EstimateCostView';
+
+ 
+export async function cost(context: vscode.ExtensionContext): Promise<void> {
+    try {
+            const writeEmitter = new vscode.EventEmitter<string>();
+            const closeEmitter = new vscode.EventEmitter<number>();
+            const pty = {
+                onDidWrite: writeEmitter.event,
+                onDidClose: closeEmitter.event,
```

Comment:
We have a terminal util [here](https://github.com/IBM-Cloud/vscode-ibmcloud-schematics/blob/main/extension-client/src/util/terminal.ts). Any particular reason this was not used here ?

Reply:
made changes & have used terminal util